edscrapers/tools/dashboard/json_parser.py

```diff
@@ -62,7 +60,7 @@ def get_total_resources_by_office(source, is_sorted=True):
         data = data['total'][source]['resources_by_office'].items()
         return dict(sorted(data, key=lambda item: item[1], reverse=True))
 
-def get_total_pages_by_office(source, is_sorted=True):
+def get_total_datasets_by_office(source, is_sorted=True):
```

Comment:
If the return value is pages, why the function name change?

Reply:
Also, this doesn't seem to be used, will flag in the code (mark for review later on).